packages/algob/src/internal/deployer.ts

```diff
@@ -669,14 +690,17 @@ export class DeployerDeployMode extends DeployerBasicMode implements Deployer {
    * @param newApprovalProgram New Approval Program filename
    * @param newClearProgram New Clear Program filename
    * @param flags Optional parameters to SSC (accounts, args..)
+   * @param appName name of the app to deploy. This name (if passed) will be used as
+   * the checkpoint "key", and app information will be stored agaisnt this name
    */
   async updateApp (
```

Comment:
Should we change parameters to object? In my view, It will more readable.
Example: updateApp({sender : master, payFlags, appID: randomAppID, ...})